Userland/Applications/ImageViewer/ViewWidget.h

```diff
@@ -66,9 +66,9 @@ class VectorImage final : public Image {
 
 class BitmapImage final : public Image {
 public:
-    static NonnullRefPtr<BitmapImage> create(Gfx::Bitmap& bitmap) { return adopt_ref(*new BitmapImage(bitmap)); }
+    static NonnullRefPtr<BitmapImage> create(Gfx::Bitmap& bitmap, Gfx::FloatPoint scale) { return adopt_ref(*new BitmapImage(bitmap, scale)); }
 
-    virtual Gfx::IntSize size() const override { return m_bitmap->size(); }
+    virtual Gfx::IntSize size() const override { return { m_bitmap->size().width() * m_scale.x(), m_bitmap->size().height() * m_scale.y() }; }
```

Comment:
Two questions about this;
1. Can the X or Y scale be negative and does ImageViewer support that?
2. A bit pedantic, but we should probably round instead of truncate here.